libs/MeshKernel/include/MeshKernel/Entities.hpp

```diff
@@ -57,14 +57,11 @@ namespace meshkernel
         {
```

Comment:
I can't comment on code that is unrelated to this PR.  GitHub does not allow it. This the closet point to where I would like to comment.

Now that we have bumped the std to c++20, can you replace  
```
template <typename T, typename std::enable_if<std::is_floating_point<T>::value>::type* = nullptr>
```
by  
```
template <std::floating_point T>
```
?